sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 11.1.0 (2021-02-11)
+
```

Comment:
Let's add 2 sections here
- Changes since 11.0.*
- Changes since the latest 11.1.0-beta

This way, people who were not using previews till now and would get this version would know what features they can expect.

You can also skip the note on the linter error fixes. This does not matter to the end user and so can be skipped